packit_service/worker/handlers/distgit.py

```diff
@@ -333,7 +333,7 @@ def run(self) -> TaskResults:
                         "Propose downstream is being retried because "
                         "we were not able yet to download the archive. "
                     )
-                    model.set_status(staus=ProposeDownstreamTargetStatus.retry)
+                    model.set_status(status=ProposeDownstreamTargetStatus.retry)
```

Comment:
:see_no_evil: funny how I did the same typos twice :grimacing:

Reply:
maybe copy paste? :D